frontend/packages/client/src/components/Proposal/VoteOptions/index.js

```diff
@@ -0,0 +1 @@
+export { default as VoteOptions } from './VoteOptions';
```

Comment:
I extracted some components from VoteOptions bc the file itself was hosting ButtonChoice, ImageBasedOptions, TextBasedOptions and in this PR I created VoteHeader. I think this will help us for the next changes on ranked vote options to have smaller files